kumascript/index.js

```diff
@@ -1,4 +1,4 @@
-const { Document, memoize } = require("../content");
+const { Document } = require("../content");
```

Comment:
Sorry. My editor pointed out that it was an import that wasn't used. I just couldn't help myself.